packages/jellyfish-api-core/src/category/spv.ts

```diff
@@ -211,3 +237,40 @@ export interface ListHtlcsOutputsResult {
   /** Object containing spent info */
   spent: SpentInfo
 }
+
+export interface CreateAnchorInput {
+  /** The transaction id of the bitcoin UTXO to spend */
+  txid: string
+  /** The output index to spend in UTXO for tx fee */
+  vout: number
+  /** Amount of output in satoshis (base unit) */
+  amount: number
+  /** WIF private key of bitcoin of signing this output */
+  privkey: string
+}
+
+export interface CreateAnchorOptions {
+  /** Send to BTC network */
+  send: boolean
+  /** Feerate (satoshis) per 1000 bytes */
+  feerate: number
```

Comment:
```suggestion
  feerate?: BigNumber
```

Reply:
don't need bignumber on feerate as it is per satoshi.. its base unit